.goreleaser.yaml

```diff
@@ -57,3 +57,106 @@ changelog:
 # Feel free to remove those if you don't want/use them.
 # yaml-language-server: $schema=https://goreleaser.com/static/schema.json
 # vim: set ts=2 sw=2 tw=0 fo=cnqoj
+nfpms:
+  # note that this is an array of nfpm configs
+  -
+    # ID of the nfpm config, must be unique.
+    # Defaults to "default".
+    id: foo
+
+    # Name of the package.
+    # Defaults to `ProjectName`.
+    package_name: daje
+
+    # Template to your app's description.
+    # Default is empty.
+    description: |-
+      daje
+      A general purpose dotfiles installer
+
+    # Your app's license.
+    # Default is empty.
+    license: Apache 2.0
```

Comment:
The app license is GPL3 and you can see it on the root of the project in the file named LICENSE